<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDisplay</class>
 <widget class="QWidget" name="SetDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_11" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_10" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_1" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_12" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
